eras/conway/impl/src/Cardano/Ledger/Conway/Rules/GovCert.hs

```diff
@@ -54,83 +54,83 @@ import Data.Word (Word8)
 import GHC.Generics (Generic)
 import NoThunks.Class (NoThunks (..))
 
-data ConwayVDelPredFailure era
-  = ConwayDRepAlreadyRegisteredVDEL !(Credential 'Voting (EraCrypto era))
-  | ConwayDRepNotRegisteredVDEL !(Credential 'Voting (EraCrypto era))
-  | ConwayDRepIncorrectDepositVDEL !Coin
-  | ConwayCommitteeHasResignedVDEL !(KeyHash 'CommitteeColdKey (EraCrypto era))
+data ConwayGovCertPredFailure era
+  = ConwayDRepAlreadyRegisteredGOVCERT !(Credential 'Voting (EraCrypto era))
+  | ConwayDRepNotRegisteredGOVCERT !(Credential 'Voting (EraCrypto era))
+  | ConwayDRepIncorrectDepositGOVCERT !Coin
+  | ConwayCommitteeHasResignedGOVCERT !(KeyHash 'CommitteeColdKey (EraCrypto era))
```

Comment:
This `GOVCERT` suffix in the data constructor seems redundant and not necessarily consistent with the rest of the predicate failures names. Any particular reason we would want it here?

Reply:
Yeah, I concur. It is inconsistent with other eras